apps/client/src/components/ConclusionPrint.js

```diff
@@ -0,0 +1,116 @@
+import React, { useContext } from "react";
+import { Paragraph, Heading, Icon } from "@datapunt/asc-ui";
+import { Alert } from "@datapunt/asc-assets";
+import uniqBy from "lodash.uniqby";
+
+import Context from "../context";
+import {
+  QuestionWrapper,
+  MainWrapper,
+  Question,
+  UserAnswer,
+  UserResult,
+  UserResultParagraph,
+} from "../pages/ResultsPageStyles";
+import { booleanOptions } from "./Question";
+
+export default ({ checker }) => {
+  const permitsPerQuestion = [];
+  const context = useContext(Context);
+  const {
+    streetName,
+    houseNumberFull,
+    postalCode,
+    residence,
+  } = context.address;
+
+  const event = new Date();
+  const options = {
+    year: "numeric",
+    month: "long",
+    day: "numeric",
+    hour: "numeric",
+    minute: "numeric",
+  };
+  const today = event.toLocaleDateString("nl-NL", options);
+
+  checker.permits.forEach((permit) => {
+    const conclusionDecision = permit.getDecisionById("dummy");
+    if (conclusionDecision.getOutput() === '"Vergunningplicht"') {
+      const decisiveDecisions = conclusionDecision.getDecisiveInputs();
+      decisiveDecisions.forEach((decision) => {
+        const decisiveQuestion = decision.getDecisiveInputs().pop();
+        const index = checker.stack.indexOf(decisiveQuestion);
+        permitsPerQuestion[index] = (permitsPerQuestion[index] || []).concat(
+          permit
+        );
+      });
+    }
+  });
+
+  return (
+    <>
+      <Paragraph fontSize={12}>{window.location.href}</Paragraph>
+
+      <Heading forwardedAs="h2">Datum</Heading>
+      <Paragraph strong>{today} uur.</Paragraph>
+
+      <Heading forwardedAs="h2">Adresgegevens</Heading>
+      <Paragraph strong>
+        {streetName} {houseNumberFull}
+        <br />
+        {postalCode} {residence}
+      </Paragraph>
+
+      <Heading forwardedAs="h2">Uw antwoorden</Heading>
+      <Paragraph>
+        Hieronder kunt u per vraag uw gegeven antwoord teruglezen.
+      </Paragraph>
+
+      <MainWrapper>
+        <Question strong>Vraag</Question>
+        <UserAnswer strong>Uw antwoord</UserAnswer>
+      </MainWrapper>
+      <div style={{ marginBottom: 40 }}>
```

Comment:
Can we make all the stuff onder this div a separate component so we can reuse it?

Reply:
Maybe include the `<MainWrapper/>` above.

Reply:
Great feedback! Fixed it